src/node/waku_node.py

```diff
@@ -84,6 +109,46 @@ def start(self, **kwargs):
             logger.error(f"REST service did not become ready in time: {ex}")
             raise
 
+    @retry(stop=stop_after_delay(5), wait=wait_fixed(0.1), reraise=True)
+    def register_rln(self, **kwargs):
+        logger.debug("Registering RLN credentials...")
+        self._docker_manager.create_network()
+        self._ext_ip = self._docker_manager.generate_random_ext_ip()
+        self._ports = self._docker_manager.generate_ports()
+        self._rest_port = self._ports[0]
+        self._tcp_port = self._ports[1]
+        self._websocket_port = self._ports[2]
+        self._discv5_port = self._ports[3]
+        self._metrics_port = self._ports[4]
+        self._api = REST(self._rest_port)
+        self._volumes = []
+
+        default_args = {
+            "rln-creds-id": None,
+            "rln-creds-source": None,
+        }
+
+        for key, value in kwargs.items():
```

Comment:
can we add a helper function ex `sanitize_docker_flags` so we don't duplicate this code?

Reply:
Function added at https://github.com/waku-org/waku-interop-tests/pull/16/commits/6040f0875dcc02f50c2b4636a7cd6becd5c81526